source/designers-guide/snippets/index.md

```diff
@@ -59,6 +59,28 @@ While snippets are mostly used to translate plain text, they are flexible enough
 {s name="frontend/checkout/cart/separate_dispatch"}<strong>bold example text</strong>{/s}
 ```
 
+### Using smarty variables and functions in snippets
+
+With smarty you have access to config variables, view variables and you can even use modifiers and functions. With this you have the possibility to make your snippets more adaptive.
+
+Here are some examples what's possible:
+
+Instead of changing the name in each snippet make use of the config variable:
+```
+Hello to the new {config name=shopName} Shop!
+```
+
+Instead of changing the year of the copyright every year, make use of a smarty modifier:
+```
+Copyright {"%Y"|strftime} {config name=shopName}
+```
+
+Determine the output of your snippets with "if-queries":
+```
+{if $sArticle.length}L{$sArticle.length} {/if}{if $sArticle.length}B{$sArticle.width} {/if}{if $sArticle.length}H{$sArticle.height} {/if}cm
```

Comment:
the if's for width and height are wrong :wink:

Reply:
You're right. Just copied it from a shop.
-> Fixed!